eo-runtime/src/main/java/EOorg/EOeolang/EOseq.java

```diff
@@ -58,25 +59,33 @@ public EOseq(final Phi sigma) {
     @Override
     public Phi lambda() {
         final Phi steps = this.take("steps");
-        final Long length = new Dataized(
-            steps.take("length")
-        ).take(Long.class);
-        for (long idx = 0; idx < length - 1; ++idx) {
-            new Dataized(
-                new PhWith(
-                    steps.take("at").copy(),
-                    0, new Data.ToPhi(idx)
-                )
-            ).take();
+        final Phi[] items = toArray(steps);
```

Comment:
@levBagryansky let's follow the convention and use static methods with class prefix: `EOseq.toArray()`
Also it would be better to rename `toArray` to something like `stepsAsArray`, `stepsToArray`. Anyway it should be a noun.

Reply:
@maxonfjvipon done